src/main/resources/fileTemplates/j2ee/Rust.rs.ft

```diff
@@ -0,0 +1,8 @@
+#**
+ * Rust Competitive Programming Template
+ *
+ * wrap the code like #[[ code ]]# so that it is generated as it is
+ *#
+#[[fn main() {
```

Comment:
start and end this template with a new line

Reply:
done